plugins/woocommerce/client/admin/config/core.json

```diff
@@ -19,13 +18,6 @@
 		"new-product-management-experience": false,
 		"onboarding": true,
 		"onboarding-tasks": true,
-		"product-variation-management": true,
-		"product-virtual-downloadable": true,
-		"product-external-affiliate": true,
-		"product-grouped": true,
-		"product-linked": true,
-		"product-pre-publish-modal": true,
-		"product-custom-fields": true,
```

Comment:
I would keep at-least these two, as first of all we are actually planning on disabled the `pre-publish` feature https://github.com/woocommerce/woocommerce/issues/47189

Also the custom fields is still relatively new, so I think we should keep the feature flag incase something big comes up.

Reply:
Good call! I just removed the commits that deleted those flags.